hs_core/templates/pages/my-resources.html

```diff
@@ -293,7 +295,7 @@ <h4 class="panel-title">
                                     Favorite
                                 {% endif %}
                             </td>
-                            <td>{{ res.updated|date:"U" }}</td>
+                            <td>{{ res.last_updated|date:"U" }}</td>
```

Comment:
Looks like I missed a last_updated, thanks for catching that.  I think it's worth making last_updated a field instead of hiding it away in metadata... helps speed things up too.  I put last_updated back to updated and got about 1 second speed up on jeff's my resource page.  I'll go ahead and create a ticket if you agree that we should add a last_updated field.  You might know something I don't.

Reply:
Yes if we add the 'last_updated' to resource class we should see performance improvement in particular when we are listing many resources (like my resource page) where for each resource we are displaying last update date. When we are accessing last_updated from metadata, we are hitting the database for each resource.

Reply:
I am now prefetching 'dates' metadata element - may be we don't need to add this additional field to resource class.